schemas/input_schema/REopt-BAU-input.json

```diff
@@ -53,9 +61,10 @@
         }
     },
     "ExistingChiller": {
-        "type": "float",
-        "required": false,
-        "description": "Only required if BAU system uses fuel for cooling. No need to define anything additional here"        
+        "fuel_cost_per_mmbtu": {
```

Comment:
There's no "fuel_cost_per_mmbtu": {} needed for ExistingChiller. If a building uses fuel for heating, it is assumed to use fuel for heating as well and the fuel cost is already in ExistingBoiler

Reply:
thanks, I removed the existing chiller in the schema